packages/docs/assets/colors.scss

```diff
@@ -0,0 +1,5 @@
+$active-blue: #2550C0;
```

Comment:
Language switching doesn't seem to work for me
![image](https://user-images.githubusercontent.com/5394573/98268093-2968e980-1f9d-11eb-94fd-07383e03fe67.png)

Reply:
It's because we don't have Russian translation yet